sympy/physics/mechanics/jointsmethod.py

```diff
@@ -0,0 +1,222 @@
+from sympy.physics.mechanics import Body, Lagrangian, KanesMethod, LagrangesMethod
+from sympy.physics.mechanics.method import _Methods
+
+__all__ = ['JointsMethod']
+
+
+class JointsMethod(_Methods):
+    """Method for formulating the equations of motion using a set of interconnected bodies with joints.
+
+    Parameters
+    ==========
+
+    body : Body or ReferenceFrame
+        The inertial(Newtonion) frame.
+    *joints : Joint
+        The joints in the system
+
+    Attributes
+    ==========
+
+    q, u : Matrix
+        Matrices of the generalized coordinates and speeds
+    bodies : iterable
+        Iterable of Body objects in the system.
+    forcelist : iterable
+        Iterable of (Point, vector) or (ReferenceFrame, vector) tuples
+        describing the forces on the system.
+    mass_matrix : Matrix
+        The system's mass matrix
+    forcing : Matrix
+        The system's forcing vector
+    mass_matrix_full : Matrix
+        The "mass matrix" for the u's and q's
+    forcing_full : Matrix
+        The "forcing vector" for the u's and q's
+    method : KanesMethod or Lagrange's method
+        Method's object.
+
+    Examples
+    ========
+
+    This is a simple example for a one degree of freedom translational
+    spring-mass-damper.
+
+    >>> from sympy import symbols
+    >>> from sympy.physics.mechanics import Body, JointsMethod, PrismaticJoint
+    >>> from sympy.physics.vector import dynamicsymbols
+    >>> c, k = symbols('c k')
+    >>> q, u = dynamicsymbols('q u')
+    >>> W = Body('W')
+    >>> B = Body('B')
+    >>> J = PrismaticJoint('J', W, B, coordinates=q, speeds=u)
+    >>> W.apply_force(c*u*W.x, reaction_body=B)
+    >>> W.apply_force(k*q*W.x, reaction_body=B)
+    >>> method = JointsMethod(W, J)
+    >>> method.form_eoms()
+    (Matrix([[-c*u(t) - k*q(t)]]), Matrix([[-B_mass*Derivative(u(t), t)]]))
+    >>> MM = method.mass_matrix_full
+    >>> forcing = method.forcing_full
+    >>> rhs = MM.LUsolve(forcing)
```

Comment:
use `method.rhs()` because this is documentation for JointsMethod not LUsolve

Reply:
As we discussed yesterday in meeting, example with `rhs` would be added in `form_eoms()`. So that user knows both methods . Also `Kanes.rhs()` uses forcing and mass_matrix instead of forcing_full, mass_matrix_full.